### ISSUE TITLE
[typescript] walkTreeData only refers to a type

### ISSUE THREAD

**Issue by f3l1x** [phphe/he-tree-vue#11]
Hi guys! Thx your this lib.

We're dealing with this kind of error in our codebase.

![image](https://user-images.githubusercontent.com/538058/79257531-099b9800-7e8a-11ea-991b-057142629f88.png)

I was trying to import also this way. It does not work neither.

```js
import * as tree from "he-tree-vue";
```

We end up with this one.

```js
// @ts-ignore
import {walkTreeData} from "he-tree-vue/dist/he-tree-vue";
```

Can you please help me? Thx.

**Comment by phphe** [comment]
This is a bug. The type declaration file should export function instead of interface. fixed in 1.2.1

**Comment by f3l1x** [comment]
Confirmed, it works. Thanks.
